aether-ui/aether/ui/assets/css/base/_modal.scss

```diff
@@ -62,7 +62,7 @@
         color: $red;
 
         &::before {
-          content: '\f06a '; // fa-exclamation-circle
+          content: '\f06a'; // fa-exclamation-circle
```

Comment:
It's possible that that whitespace was there intentionally... @sabeen ?

Reply:
I check both cases, with and without whitespace and the result it's the same, there is the `margin-right` option that makes the whitespace useless.